templates/default/knative-istio.yml.erb

```diff
@@ -66,6 +66,12 @@ spec:
         protocol: HTTP
       hosts:
         - "*"
+    - port:
+        number: 9010
```

Comment:
Was `9010` chosen randomly? does it have to match with another port somewhere else?